src/test/components/TrackScreenshots.test.js

```diff
@@ -84,13 +98,13 @@ describe('timeline/TrackScreenshots', function() {
   it('makes sure the hover image does not go off the left side of screen', () => {
     const { moveMouseAndGetLeft } = setup();
     const pageX = LEFT;
-    expect(moveMouseAndGetLeft(pageX) >= 0).toBe(true);
+    expect(moveMouseAndGetLeft(pageX)).toBe(0);
   });
 
   it('makes sure the hover image does not go off the top side of screen', () => {
     const { moveMouseAndGetTop } = setup();
     const pageX = LEFT;
-    expect(moveMouseAndGetTop(pageX) >= 0).toBe(true);
+    expect(moveMouseAndGetTop(pageX)).toBe(0);
```

Comment:
I didn't delete this, modified instead. If you want I'll remove.

Reply:
My suggestion was to remove the previous one `makes sure the hover image does not go off the **left** side of screen`, but only if similar test was moved to the previous `moves the hover when moving the mouse`.

But actually this is maybe better to test that here. Just we need to test it a bit more:
So with a screenshot width of width=262.50px, we should have left == 0 until hovering at 131px. Then at 132px this should start moving (maybe that's 133px because of some rounding).
Ideally we should test these 2 values, to show this threshold.

(and please add comments explaining these values too)

Reply:
I removed `makes sure the hover image does not go off the **left** side of screen` because the logic seems to be as `moves the hover when moving the mouse` as you said.

Reply:
Yeah, but now we're missing a test that tests this. I don't mind whether this is in `moves the hover when moving the mouse` or in a new test block, but we need some test that says that while we move the mouse between 100 and 150, left stays at 0.

So I'd do this:
```js
  it('makes sure the hover image does not go off the left side of screen', () => {
    const { moveMouseAndGetLeft } = setup();

    // Because the hover screenshot's size is 300px, it's stuck at the
    // left of the window when the mouse is between 100 (the left of
    // the track) and 150px.
    expect(moveMouseAndGetLeft(100)).toBe(0);
    expect(moveMouseAndGetLeft(120)).toBe(0);
    expect(moveMouseAndGetLeft(150)).toBe(0);

    // Starting at 151px, the hover tooltip starts to move.
    expect(moveMouseAndGetLeft(151)).toBe(1);
  });
```

What do you think?